server/src/main/java/org/opensearch/cluster/coordination/Coordinator.java

```diff
@@ -1735,6 +1741,7 @@ public void onSuccess(String source) {
                                     logIncompleteNodes(Level.WARN);
                                 }
                                 cancelTimeoutHandlers();
+                                closeElectionScheduler();
```

Comment:
what happens during failure? do we keep the scheduler open?

Reply:
We keep the scheduler open during failure so as to track the failed election attempts. If another node becomes the leader then the scheduler will close on current node when it becomes follower (on publication of cluster state to this node).